book/src/testnet-participation.md

```diff
@@ -29,6 +29,17 @@ traversal issues.  A cloud-hosted machine works best.  **Ensure that IP ports
 Prebuilt binaries are available for Linux x86_64 (Ubuntu 18.04 recommended).
 MacOS or WSL users may build from source.
 
+For a performance testnet with many transactions we have some preliminary recomended setups:
+| | Low end | Medium end | High end |
+| --- | ---------|------------|----------|
+| CPU | AMD Threadripper 1900x | AMD Threadripper 2920x | AMD Threadripper 2950x |
+| RAM | 16GB | 32GB | 64GB |
+| OS Drive | Samsung 860 Evo 2TB | Samsung Evo 1TB | Samsung 970 Pro 1TB |
```

Comment:
Why 2TB OS drive for low end but 1TB for the other two?

Reply:
The OS drive in the low end has to store OS + ledger/accounts, but the other ones only need to store the OS and whatever programs the machine needs.

Reply:
there are 2TB 970's now.  we should push fo 4TB of state space

Reply:
it might be doable to put 3x 2080ti and 3x970 2tb.  we need to test it out

Reply:
https://www.tweaktown.com/news/65736/samsung-2tb-970-pro-scheduled-release/index.html

Reply:
No 2TB 970 pros afaict

Reply:
Evo

Reply:
970 Pro has a 2x speed and write endurance per GB advantage over Evo. Do we need to give that up? Evo generally uses cheap nand that goes bad faster and can't withstand heavy usage so hard to recommend for a server. Although the warranty is the same. Seems that 970 evo does beat 860 pro generally, so maybe we can go with that for the OS drive.